src/asciidoc/docs/blueprints/customVCC.adoc

```diff
@@ -0,0 +1,208 @@
+
+[source,json]
+----
+include::../_includes/vcc-blueprint-settings.json[tags=configuration]
+----
+
+=== Add the template logic
+Import the packages.
+
+[source,javascript]
+----
+import CustomVCC from '@withkoji/custom-vcc-sdk';
+----
+
+Initialize `CustomVCC` and set a listener to load the Koji theme.
+This listener allows you to style the VCC to match the user’s current theme on Koji (dark or light).
+
+.React
+[source,javascript]
+----
+include::../_includes/customVCCReact.js[tags=constructor,indent=0]
+----
+
+.Vanilla JS
+[source,javascript]
+----
+include::../_includes/customVCCVanilla.js[tags=initTheme,indent=0]
+----
+
+Register your custom VCC to connect to Koji and set a listener to update your application state when the custom VCC updates.
+
+.React
+[source,javascript]
+----
+include::../_includes/customVCCReact.js[tags=mount,indent=0]
+----
+.Vanilla JS
+[source,javascript]
+----
+include::../_includes/customVCCVanilla.js[tags=registerVCC,indent=0]
+----
+
+Make a call to the API endpoint to retrieve the required values.
+
+.React
+[source,javascript]
+----
+include::../_includes/customVCCReact.js[tags=loadAPI,indent=0]
+----
+
+.Vanilla JS
+[source,javascript]
+----
+include::../_includes/customVCCVanilla.js[tags=loadAPI,indent=0]
+----
+
+Create a setter to return the selected value from your application back to the VCC on Koji.
+
+Use `customVCC.change()` to notify Koji that the VCC value has been updated and `customVCC.save()` to tell Koji to commit the change to the remixer's VCC.
+
+.React
+[source,javascript]
+----
+include::../_includes/customVCCReact.js[tags=selectValue,indent=0]
+----
+.Vanilla JS
+[source,javascript]
+----
+include::../_includes/customVCCVanilla.js[tags=selectValue,indent=0]
+----
+
+Use the Koji theme and the API results to render a drop-down menu that allows users to select a value.
+
+.React
+[source,javascript]
+----
+include::../_includes/customVCCReact.js[tags=render,indent=0]
+----
+
+.Vanilla JS
+[source,javascript]
+----
+include::../_includes/customVCCVanilla.js[tags=render,indent=0]
+----
+
+=== Add the styles
+Define styles for the template, using the theme values to match the user's Koji theme.
+
+.React
+[source,javascript]
+----
+include::../_includes/customVCCReact.js[tags=styles,indent=0]
+----
+
+.Vanilla JS
+[source,css]
+----
+include::../_includes/vcc-blueprint-styles.css[]
+----
+
+=== Test
+
+To save development time, Koji provides an easy way to test a custom VCC before it is published. First you will need to create a test consumer project, do so by remixing https://withkoji.com/templates/seane/simple-react-scaffold-remix-[Simple React Scaffold].
+
+. In your custom VCC project, click the Remote tab in the right pane.
+. Click Copy URL to get the staging URL for your project.
+. Open your consumer project.
+. In your consumer project, open the Strings customization file, and click Code to view the raw JSON.
```

Comment:
```suggestion
. In your consumer project, open a JSON customization file, and add or edit a field to the custom VCC.
+
The type must match this format, using the URL you just copied as YOUR-URL:
+
`"type": "custom<YOUR-URL>"`
```
I'm trying to make these instructions a bit more generic vs. the tutorial, which is very specific. But, you might need to play around with the steps and language a bit to make it read more clearly when it's all in sequence.